maven/config/pom.xml

```diff
@@ -9,7 +9,7 @@
     <engine.page.url>https://product.ivyteam.io</engine.page.url>
     <flatten.plugin.version>1.7.0</flatten.plugin.version>
     <project.build.plugin.version>13.1.1-SNAPSHOT</project.build.plugin.version>
-    <web.tester.version>12.0.2-SNAPSHOT</web.tester.version>
+    <web.tester.version>13.1.1-SNAPSHOT</web.tester.version>
```

Comment:
@ivy-rew Why 13.1.1? And not 13.2.0?

Reply:
wouldn't fit for the release/13.1 branch, which isn't released yet.
usually we run this job again after the official release branch is here. Though this hasn't be done after the lts12 release (maybe because we moved the 13 branch again later on)